examples/cpp/configuration/PublisherApp.cpp

```diff
@@ -268,8 +268,8 @@ bool PublisherApp::publish()
     std::unique_lock<std::mutex> matched_lock(mutex_);
     cv_.wait(matched_lock, [&]()
             {
-                // at least one has been discovered
-                return ((matched_ > 0) || is_stopped());
+                // at least wait_ entities discovered. 
```

Comment:
Please rewrite the comment to improve readability. 
```suggestion
                // Publisher starts sending messages when enough entities have been discovered.
```